### PR TITLE
02 git feature rock paper scissors

### PR DESCRIPTION
**Pull request by SergeyChekanov**
Refactor task is done.

________________________ Refactor Task__________________________________________
Please refactor (or add some commits) so the branch structure became e.g.:

1) dummy (getting choice from user) and makefile
2)random generation
3)comparison and winner determination.

### CODE REVIEW

File: 02-Git/Dummy_Program.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+int main(void)
+{
+	printf("Hello rock_paper_scissors_game!\n");
+	getchar();
+	return 0;
+}
```
Comment (an1kh) [inline 02-Git/Dummy_Program.c]: No empty line at the end of the file

File: 02-Git/Game.c [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#include <stdio.h>
+#include <iostream.h>
+#include <string>
+#include <time.h>
+//#include <stdio.h>
```
Comment (an1kh) [inline 02-Git/Game.c]: Why this is needed?

File: 02-Git/Game.c [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+int main()
+{
+    setlocale(LC_ALL, "Russian");
+
+    srand(static_cast<unsigned>(time(NULL)));
+
+    int user_wins = 0;
+    int comp_wins = 0;
+
+        int cc = rand() % 3;
```
Comment (an1kh) [inline 02-Git/Game.c]: Double indentation

File: 02-Git/makefile [hunk 4]
```diff
@@ -0,0 +1,6 @@
+dummy.exe:	Dummy_Program.o
+	gcc -o dummy.exe Dummy_Program.o
+ 
+Dummy_Program.o:	Dummy_Program.c
+	gcc -O0 -g3 -Wall -c Dummy_Program.c
+	gcc -masm=intel -g3 -Wall -c Dummy_Program.c -S
```
Comment (an1kh) [inline 02-Git/makefile]: No empty line at the end of the file

File: 02-Git/makefile [hunk 5]
```diff
@@ -1,6 +1,9 @@
...
 	gcc -o dummy.exe Dummy_Program.o
+	gcc -o Game.exe Game.o
  
 Dummy_Program.o:	Dummy_Program.c
 	gcc -O0 -g3 -Wall -c Dummy_Program.c
-	gcc -masm=intel -g3 -Wall -c Dummy_Program.c -S
\ No newline at end of file
+	gcc -masm=intel -g3 -Wall -c Dummy_Program.c -S
+	gcc -O0 -g3 -Wall -c Game.c
+	gcc -masm=intel -g3 -Wall -c Game.c -S
```
Comment (an1kh) [inline 02-Git/makefile]: No empty line at the end of the file

File: 02-Git/Game.c [hunk 6]
```diff
@@ -1,8 +1,7 @@
-#include <stdio.h>
-#include <iostream.h>
+#include <iostream>
 #include <string>
 #include <time.h>
-//#include <stdio.h>
+#include <stdio.h>
```
Comment (an1kh) [inline 02-Git/Game.c]: Why the includes were changes?
IMO it is needed to set themproperly in initial commit.

**Review by an1kh — CHANGES_REQUESTED** [review]
Please fix the issues found.

File: 02-Git/Dummy_Program.c (hunk 1 above)
Comment (SergeyChekanov) [reply 02-Git/Dummy_Program.c]: Add empty line at the end of the file

File: 02-Git/Game.c (hunk 2 above)
Comment (SergeyChekanov) [reply 02-Git/Game.c]: Delete temp work with #include <stdio.h>

File: 02-Git/Game.c (hunk 3 above)
Comment (SergeyChekanov) [reply 02-Git/Game.c]: Fixed

File: 02-Git/makefile (hunk 4 above)
Comment (SergeyChekanov) [reply 02-Git/makefile]: Fixed

File: 02-Git/Game.c (hunk 6 above)
Comment (SergeyChekanov) [reply 02-Git/Game.c]: Delete temp work with #include <stdio.h>, was problem with build.

File: 02-Git/makefile (hunk 5 above)
Comment (SergeyChekanov) [reply 02-Git/makefile]: Add empty line

File: 02-Git/Game.c [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+        int cc = rand() % 3;
+
+    }
+
+    cout << endl << "Нажмите любую клавишу...";
+    _getch();
+    return 0;
+}
```
Comment (an1kh) [inline 02-Git/Game.c]: No empty line at the end of the file

File: 02-Git/Game.c [hunk 8]
```diff
@@ -24,7 +24,7 @@ int main()
 
     int user_wins = 0;
     int comp_wins = 0;
-    
+
```
Comment (an1kh) [inline 02-Git/Game.c]: Do this in commit where these trailing spaces were added

**Review by an1kh — CHANGES_REQUESTED** [review]
Few minor issues remain.
Also please do not add "Fix: ....." to commit description. In the current project we do not need such history and need only final commits.

File: 02-Git/Game.c (hunk 7 above)
Comment (ChekanovSerhii) [reply 02-Git/Game.c]: Add empty line in last commit.

**Review by an1kh — APPROVED** [review]
Well done! Please merge

File: 02-Git/Game.c (hunk 8 above)
Comment (SergeyChekanov) [reply 02-Git/Game.c]: Fixed

**Pull request merged**